docs/flow/modin/experimental/core/execution/native/implementations/hdk_on_native/index.rst

```diff
@@ -1,7 +1,12 @@
 :orphan:
 
-OmnisciOnNative execution
-=========================
+HdkOnNative execution
```

Comment:
We will need to change everything accordingly in docs/flow/modin/experimental/core/execution/native/implementations/hdk_on_native but this can be done as a separate step

Reply:
I am okay with a different PR but I am rather we put it in 0.16.

Reply:
It could take time to change all developer level docs. we can do this for, say, 0.16.1 and then backport

Reply:
We have time until we support pandas 1.5 so I suggest working on this now.

Reply:
Maybe for now we should put some note here (and probably in other files if it makes sense) that the documentation doesn't fully correspond the code as OmniSci has been replaced with HDK?

Reply:
Yes, agree

Reply:
...and will be fixed in the next release smth like this also I think

Reply:
fine by me